GUI/app.py

```diff
@@ -70,8 +73,10 @@ def printdata(entrybox):
         
         e.pack()            
         area.mainloop()
-        
+     # get_depth functon takes the initial depth of the dustbin
     def get_depth(self,setup):
+        
```

Comment:
get_depth function used to get the data from the sensor.  we pass either True or false .Passing True  gets initial setup. Passing False takes the   current depth.